generate-groups.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
```

Comment:
Could all the scripts be placed in `scripts` directory as root is getting a tad large? I tried this concept in https://github.com/openSUSE/osc-plugin-factory/pull/1177/files#diff-b67911656ef5d18c4ae36cb6741b7965R25, preferably prefix with `osrt-` for basic namespacing.

I can merge that request today, but the `Makefile` change to install them will mean the `spec` needs to be updated to own the files. I plan to make a packagelist subpackage which should likely own them.

Reply:
script or scripts? Do you want to merge the cited request first so I can rebase on it?

Reply:
`script`, yeah I just did.

Reply:
Not sure if we want to go with multiple directories, but since I was installing them in path I went with `osrt-$tool-...`, but as long as we have them it git it's a step forward.